Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPCorefinementVisitor.h

```diff
@@ -58,15 +58,15 @@ class PMPCorefinementVisitor{
   /// \param i_id the id of the intersection point, starting at 0. Ids are consecutive.
   /// \param sdim indicates the dimension of the simplex part of the face that is intersected by the edge
   ///             (0 for a vertex, 1 for an edge, 2 for the interior of the face)
+  /// \param tm_e mesh containing `h_e`
+  /// \param tm_f mesh containing `h_f`
+  /// \param h_e a halfedge from `tm_e`
```

Comment:
Order is still a bit strange first description of `tm_*` and afterwards `h_*` whilst in the argument lists it is the other way around.